pages/lists/list-user/web-space/de.text

```diff
@@ -0,0 +1,13 @@
+@title = 'Web Space'
+(Gemeinsame Dokumente)
+
+h2. Wie kann ich eine Datei hochladen oder ansehen?
+
+Vollständige Informationen dazu auf [[shared-web-space]].
+
+h2.  Warum gelingt es mir nicht, eine Datei hochzuladen?
```

Comment:
the excess space should be removed (just as in the original)

Reply:
Done